Package.swift

```diff
@@ -16,11 +16,16 @@ let package = Package(
     .library(name: "_CAsyncSequenceValidationSupport", type: .static, targets: ["AsyncSequenceValidation"]),
     .library(name: "AsyncAlgorithms_XCTest", targets: ["AsyncAlgorithms_XCTest"]),
   ],
-  dependencies: [.package(url: "https://github.com/apple/swift-collections.git", .upToNextMajor(from: "1.0.3"))],
+  dependencies: [.package(url: "https://github.com/apple/swift-collections.git", .upToNextMajor(from: "1.0.4"))],
   targets: [
     .target(
       name: "AsyncAlgorithms",
-      dependencies: [.product(name: "Collections", package: "swift-collections")]
+      dependencies: [.product(name: "Collections", package: "swift-collections"),],
+      swiftSettings: [
+        .unsafeFlags([
+          "-strict-concurrency=complete"
+        ], .when(configuration: .debug))
```

Comment:
I don't think we add this. AFAIK any package with `unsafeFlags` can't be depended on.

Reply:
ugh that thing is still present eh... that is unfortunate.

Reply:
FWIW - I've been shimming around this by only setting it when there's an environment variable ('CI') set so that I can still get the benefit in my local CI without impacting downstream packages.

Reply:
@neonichu is there a sanctioned way to have strict concurrency warnings on without negatively impacting downstream consumers?

Reply:
We have a discussion about this in the Swift repo https://github.com/apple/swift/pull/65993. It would be great if we can find a solution to safely opt-in to strict checking . cc @tomerd @hborla

Reply:
It seems to me that it is impossible to safely opt-in to a flag that is considered to be unstable. 

I would suggest working around this by conditionally passing the unsafe flag based on some environment variable so that it can be enabled for local development / CI, but not for clients of the package.